### ISSUE TITLE
Launcher sends invalid ubjson message

### ISSUE THREAD

**Issue by altf4** [project-slippi/slippi-desktop-app#55]
When the user hasn't yet selected whether a Console connection "is realtime" yet or not, the value is null. So when it gets serialized into ubjson and sent to console, the ubjson looks like this: (just a partial look)

 0x69, 0x0a, 0x69, 0x73, 0x52, 0x65, 0x61, 0x6c, 0x74, 0x69, 0x6d, 0x65, 0x4e

That last 0x4e (N) is a ubjson "Null operation" and has no value. So the whole message becomes invalid to parse. 

Basically, the app should default "isRealtime" to false, rather than leaving it null when unchosen.
https://github.com/project-slippi/slippi-desktop-app/blob/a8e783cab96cf51831850c092b848c323005e854/app/domain/ConsoleCommunication.js#L74
